core/migrations/0033_greatmedia.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.14 on 2020-09-16 09:18
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailmedia', '0003_copy_media_permissions_to_collections'),
+        ('core', '0032_auto_20200908_1151'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GreatMedia',
+            fields=[
+                ('media_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailmedia.Media')),
+                ('transcript', models.TextField(blank=True, null=True, verbose_name='transcript')),
```

Comment:
Tiny change with a capital letter - see comment on the model below

```suggestion
                ('transcript', models.TextField(blank=True, null=True, verbose_name='Transcript')),
```